crates/cxx-qt-build/src/lib.rs

```diff
@@ -939,7 +939,7 @@ impl CxxQtBuilder {
                 // in the same folder as the CMakeLists where the qt_add_qml_module is called
                 //
```

Comment:
all the exported files do not get regenerated if you just delete the `cxxqt/qml_modules/` folder. Should there be a rerun-if-changed somewhere? :thinking: Or would that cause problems as parent crates would then cause child ones to rebuild, so you would forever be rebuilding.

Maybe we just don't support manually cleaning that folder without also cleaning the normal build folders?